modules/tinymce/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `transparent` property for `iframe` dialog component #TINY-8534
 - New `removeAttributeFilter` and `removeNodeFilter` functions to the `DomParser` and DOM `Serializer` APIs #TINY-7847
 - New `iframe_template_callback` option in the `media` plugin. Patch provided by Namstel #TINY-8684
+- Support annotating block elements directly when using the editor annotator #TINY-8698
```

Comment:
Should this be in `Improved`? Also we aren't supporting all block elements, only specific ones so we probably need to be a little clearer about that...

Reply:
Yeah, I was tossing between the two when creating the entry. I think if it supported all blocks or was more general then maybe it would make more sense to be in `Added`. Although, given it is quite specific and not integrator configurable, then `Improved` might be safer/better.

I am happy to move to `Improved` and fix the message to say certain blocks.

Reply:
I have moved it over